src/prob/prob_init_fluid.cpp

```diff
@@ -481,7 +460,8 @@ void incflo::init_abl (Box const& vbx, Box const& gbx,
               theta = m_temperature_values_d[t] + (z-m_temperature_heights_d[t])*slope;
             }
         }
-        
+#if 0 
+ figure out how to get on device       
```

Comment:
Regarding this, since this is a one-time initialization just do it on host and push it to device? That's what we do in nalu-wind

Reply:
good idea, I'll fix this on a different PR